README.md

```diff
@@ -8,12 +8,24 @@ The BPMN linter used by the Camunda Desktop and Web Modeler. Batteries included.
 * configures linter based on `modeler:executionPlatform` and `modeler:executionPlatformVersion`
 * creates error messages to be shown in desktop and web modeler
 * creates errors to be shown in properties panel
+* creates error overlays to be shown on canvas 
 
 # Usage
 
 ```javascript
+import Modeler from 'bpmn-js/lib/Modeler';
+
 import { Linter } from '@camunda/linting';
-import { getErrors } from '@camunda/linting/properties-panel';
+
+import lintingModule from '@camunda/linting/modeler';
```

Comment:
Will we still include that in the Camunda Modeler or now rather in `camunda-bpmn-js`?

Reply:
This will have to be integrated into the Desktop and Web Modeler. It's not part of `camunda-bpmn-js`.